libs/books/data-access/src/lib/+state/books.actions.ts

```diff
@@ -6,12 +6,12 @@ export const searchBooks = createAction(
   props<{ term: string }>()
 );
 
-export const searchBooksSuccess = createAction(
+export const searchBooksCompleted = createAction(
```

Comment:
I disagree about the use of `Completed` vs `Success`. Something could complete and still have failed.